codespeed/models.py

```diff
@@ -73,6 +76,31 @@ def save(self, *args, **kwargs):
         super(Project, self).save(*args, **kwargs)
 
 
+class HistoricalValue(object):
+    def __init__(self, name=None, val=0, color='none'):
+        self.name = name
+        self.val = val
+        self.color = color
+
+    def if_significant_update_to(self, val, color, name):
```

Comment:
I agree, but what about 'update_if_less_important_than'? It reads pretty well at the point of usage:

    max_trend.update_if_less_important_than(val, color, benchmark_name)